static/app/views/issueDetails/metricIssuesSection.tsx

```diff
@@ -0,0 +1,156 @@
+import {Fragment, useMemo} from 'react';
+import moment from 'moment-timezone';
+
+import {LinkButton} from 'sentry/components/button';
+import {DateTime} from 'sentry/components/dateTime';
+import {t} from 'sentry/locale';
+import type {Event} from 'sentry/types/event';
+import type {Group} from 'sentry/types/group';
+import type {Organization} from 'sentry/types/organization';
+import type {Project} from 'sentry/types/project';
+import {useApiQuery} from 'sentry/utils/queryClient';
+import {useLocation} from 'sentry/utils/useLocation';
+import type {TimePeriodType} from 'sentry/views/alerts/rules/metric/details/constants';
+import RelatedIssues from 'sentry/views/alerts/rules/metric/details/relatedIssues';
+import RelatedTransactions from 'sentry/views/alerts/rules/metric/details/relatedTransactions';
+import {
+  Dataset,
+  type MetricRule,
+  TimePeriod,
+} from 'sentry/views/alerts/rules/metric/types';
+import {extractEventTypeFilterFromRule} from 'sentry/views/alerts/rules/metric/utils/getEventTypeFilter';
+import {isCrashFreeAlert} from 'sentry/views/alerts/rules/metric/utils/isCrashFreeAlert';
+import {SectionKey} from 'sentry/views/issueDetails/streamline/context';
+import {InterimSection} from 'sentry/views/issueDetails/streamline/interimSection';
+
+interface MetricIssuesSectionProps {
+  event: Event;
+  group: Group;
+  organization: Organization;
+  project: Project;
+}
+
+export default function MetricIssuesSection({
+  organization,
+  event,
+  group,
+  project,
+}: MetricIssuesSectionProps) {
+  const location = useLocation();
+  const ruleId = event.contexts?.metric_alert?.alert_rule_id;
+  const {data: rule} = useApiQuery<MetricRule>(
+    [
+      `/organizations/${organization.slug}/alert-rules/${ruleId}/`,
+      {
+        query: {
+          expand: 'latestIncident',
```

Comment:
didn't notice this last time, but can probably be dropped unless we're using rule.latestIncident somewhere on this page

Reply:
also you might consider making this a shared hook so requests aren't duplicated across a single issue details page